pkg/porter/build.go

```diff
@@ -54,16 +57,8 @@ func (p *Porter) Build(opts BuildOptions) error {
 		return errors.Wrap(err, "unable to generate manifest")
 	}
 
-	// Publish may invoke this method and the manifest will already be
-	// populated.  Only load if still empty.
-	// For instance, Publish may be called with a full, new bundle reference
-	// via --tag, which will update the invocation image name and spark a new
-	// build here.  If we re-load from the local manifest, we will lose these
-	// values.
-	if p.Manifest == nil {
-		if err := p.LoadManifestFrom(build.LOCAL_MANIFEST); err != nil {
-			return err
-		}
+	if err := p.LoadManifestFrom(build.LOCAL_MANIFEST); err != nil {
```

Comment:
What is the flow when someone does `publish --reference`. Do we do a full rebuild, or do we only retag? I'm wondering why it's safe now to unconditionally load the generated manifest.

I'm not you asking to change anything, just making sure I understand the new workflow with these tags.

Reply:
Good question.  Now, when a `--reference` override is supplied, we only retag (invocation image and final bundle reference).  This does mean the bundle.json and porter.yaml files in the invocation image will be from the unchanged internal manifest.

Reply:
Can you  document this new "re-tag not rebuild" behavior in the `LongDescription` of the `porter publish`  command?

Reply:
👍 Added.